docs/analytics/CAR-2019-07-001/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "CAR-2019-07-001: Access Permission Modification"
+layout: analytic
+submission_date: 2019/07/08
+information_domain: Host
+subtypes: File
+analytic_type: Situational Awareness
+contributors: Meric Degirmenci, MITRE
+---
+
+Adversaries sometimes modify object access rights at the operating system level. There are varying motivations behind this action -  they may not want some files/objects on systems to be persistent and provide admin only rights; also, they may want files to be accessible with lower levels of permissions.
```

Comment:
Re phrase in line 2

Adversaries sometimes modify object access rights at the operating system level. There are varying motivations behind this action -  they may not want some files/objects to be be changed on systems for persistency reasons and provide admin only rights; also, they may want files to be accessible with lower levels of permissions.
Write